Documentation/using-docker-skim.md

```diff
@@ -0,0 +1,77 @@
+# Using docker-skim
+
+**Note: running the docker daemon with this tool is not supported and may break with no notice**.
+
+## Using docker-skim to run docker 1.12
+
+On a Container Linux machine, setup the following systemd unit for docker.
+Override the existing unit by placing it at the path
+`/etc/systemd/system/docker.service`.
+
+If this is done on a running machine, rather than during provisioning via ignition, first stop the existing docker and containerd units.
+
+```ini
+[Unit]
+Description=Docker Application Container Engine
+Documentation=http://docs.docker.com
+After=docker.socket
+Requires=docker.socket
+
+[Service]
+Type=simple
+
+ExecStart=/usr/bin/rkt run --dns=host --interactive \
+  --insecure-options=image \
```

Comment:
This line should be removed before merging; the signatures aren't available yet due to complications.